docs/arc42/deployment_view.md

```diff
@@ -0,0 +1,283 @@
+# Deployment View
+
+The deployment view describes how the geOrchestra Gateway is deployed in production environments and the infrastructure required to support it.
+
+## Overview
+
+The geOrchestra Gateway is designed to be deployed as a containerized application, typically using Docker. It is part of the broader geOrchestra Spatial Data Infrastructure (SDI) ecosystem and acts as the central entry point for all user requests to geOrchestra services.
```

Comment:
Could cause friction in the community. Some people don't use docker at all and will feel excluded if we keep this statement. Since this is a spring boot app, it doesn't rely on docker. I suggest to reformulate. Maybe something like
"The geOrchestra Gateway is designed as a Spring Boot microservice, that can be run as a simple executable, a system service or be deployed as a containerized application, typically using Docker"

Reply:
How sensitive. How about this:

```
The geOrchestra Gateway is designed as a Spring Boot microservice that can be run
as a simple executable, a system service, or be deployed as a containerized application
(typically using Docker). It is part of the broader geOrchestra Spatial Data Infrastructure (SDI)
ecosystem and acts as the central entry point for all user requests to geOrchestra services.
```

* [Fix](https://github.com/georchestra/georchestra-gateway/compare/238db3b807f84932d4130ff5ff556dde2be8bfaf..7f3d91e2d65fe257262e34db940def6fef69bd7f)
* [Live](https://groldan.github.io/georchestra-gateway/arc42/deployment_view/#overview)
  *  [Deployment options](https://groldan.github.io/georchestra-gateway/arc42/deployment_view/#deployment-options)

Reply:
Perfect, thanks ! ;o)

Reply:
a bit late to the party, but i second that version, thanks !